client/src/pages/private/SiteTable.js

```diff
@@ -168,7 +168,13 @@ export default ({ sites, viewOnly }) => {
     );
   }, [roles]);
 
-  const sort = (array) => _orderBy(array, [orderBy, 'operatorName'], [order]);
+  const sort = (array) => {
```

Comment:
Do we need this sort at all? I would think we should be handling sorting on the back end in most cases, sites are surely sorted from the API now right?

I guess this component isn't doing API pagination or anything so it might require a bit of a rewrite to properly pass in pagination values from the back end. If we can't fit it into this PR that would be a great enhancement in the future

Reply:
In the case that we can't do server side sortation, I vote we get rid of lodash's orderBy and write our own that will function a bit more like we want, this solution is mostly fine but it won't sort uppercase and lowercase letters perfectly, ie:

With this method:
`a, A, a => a, A, a`

When it should be  
`a, A, a => a, a, A`

It's a pretty low priority change so don't consider this a blocker but I'd recommend writing a function that makes use of `localeCompare`, something like

```
/**
 * Sorts a list of objects based on the values of a given key
 * 
 * @param {*} items list of objects to sort
 * @param {*} key object key to sort by
 * @param {*} direction asc or desc, will not sort if the value is anything else
 * @returns sorted list of objects based on given key and direction
 */
const orderBy = (items, key, direction) => {
  return items.sort((itema, itemb) => {
    switch (direction) {
      case 'asc':
        return itema[key].localeCompare(itemb[key], 'en', { numeric: true });
      case 'desc':
        return itemb[key].localeCompare(itema[key], 'en', { numeric: true });
      default:
        return 0;
    }
  });
};
```

Reply:
Nit: I don't love this function name

Reply:
We do need the sort function, it's what's called on the FE when the user sorts by clicking on a column (participants table gets sorted on BE but sites table and others get sorted on FE). 
I will get rid of the lodash function. It seems silly to use a library function when it doesn't even do what we need.
Pasting your function worked so I'm not going to reinvent the wheel, but I'm renaming to avoid conflicts with the 'orderBy' variable name that we use everywhere with it.